fixed_frame: ""
object_frame: ""
results_directory: ""
pcd_filename: ""
get_avg_neighbor_count: false
compare_dbs: []
visualize_results: true

optimization:
  radius: 0.4
  max_steps: 10
  step_improvement_threshold: 0.01

ik_solver_config:
  name: ""

display_config:
  name: ""
